src/layout/PaymentDetails/PaymentDetailsComponent.tsx

```diff
@@ -20,6 +20,11 @@ export function PaymentDetailsComponent({ node }: IPaymentDetailsProps) {
   const mappedValues = FD.useMapping(mapping);
   const prevMappedValues = useRef<Record<string, unknown>>(mappedValues);
```

Comment:
The previous value is currently initialized to the current value, so the existing use effect will not fire on mount. Would it work if you just initialized `prevMappedValues` as `undefined` instead?